settings.gradle.kts

```diff
@@ -7,10 +7,14 @@ dependencyResolutionManagement {
 }
 
 pluginManagement {
+    repositories {
+        google()
```

Comment:
Also needs mavenCentral() here, otherwise fails with 
`Could not GET 'https://jcenter.bintray.com/....'. Received status code 502 from server: Bad Gateway`

Reply:
Heya!

Could you try with the version up on `main` right now?

I've addressed these and some other changes there some time ago. For example, here's `mavenCentral()` on `settings.gradle.kts`: https://github.com/g0dkar/qrcode-kotlin/blob/main/settings.gradle.kts#L3-L5

If it still doesn't work, please let me know! :D

Reply:
It is strange, how does it work on your side?

For me it fails on `Plugin [id: 'com.android.library', version: '7.2.1', artifact: 'com.android.tools.build:gradle:7.2.1'] was not found in any of the following sources` too.

Also the `mavenCentral` you mentioned is for the project dependencies, not for plugins

Reply:
Ah, I see!

Everything is working here 🤔 

I think I know why it is working here tho: some shenanigans I have to have in place for my work. Let me comment those and try again!

Thanks for bringing this into light! <3

Reply:
Okay, it WAS the stuff I had in place for work 🤦 

I'm committing the fixes right now! Thanks again for reporting on this!

Reply:
Okay, I just sent some fixes for this. Could you try again, please? 🙇